index.html

```diff
@@ -352,6 +353,19 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
                 </div>
             </section>
 
+            <!-- Location -->
+            <section id="location" class="wrapper style1 special fade-up">
+                <div class="container">
+                    <header class="major">
+                        <h2>Map</h2>
+                    </header>
+
+                    <div id="map"></div>
+
+                    <p><a href="http://www2.mmu.ac.uk/media/mmuacuk/content/documents/how-to-find-us/access-guides/Access-Guide-Business-School.pdf" target="_blank">Access guide</a>,
+                        <a href="https://www2.mmu.ac.uk/media/mmuacuk/content/documents/how-to-find-us/Manchester-Campus-Map-A4.pdf" target="_blank">Campus map</a></p>

```

Comment:
Nit: remove one space from the second "a href" tag so that they are aligned or consider having them ordered like:
```
<p>
    <a href></a>,
    <a href></a>
</p>
```